jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -254,4 +273,10 @@ Terminal.propTypes = {
   })
 };
 
+// Define jujushell API operations and codes.
```

Comment:
these constants should probably be defined at the top of the file.

Reply:
I am not sure, they are not exported nor important, so giving them more precedence over important stuff seems arbitrary.